ffv1.md

```diff
@@ -443,57 +455,57 @@ R~i~
 : the Range at the i-th symbol.
 
 r~i~
-: the boundary between two sub-ranges of R~i~: a sub-range of r~i~ values and a sub-range R~i~ - r~i~ values.
+: the boundary between two subranges of R~i~: a subrange of r~i~ values and a subrange R~i~ - r~i~ values.
 
 L~i~
 : the Low value of the Range at the i-th symbol.
 
 l~i~
-: a temporary variable to carry-over or adjust the Low value of the Range between range coding operations.
+: a temporary variable to carry over or adjust the Low value of the Range between range coding operations.
 
 t~i~
-: a temporary variable to transmit sub-ranges between range coding operations.
+: a temporary variable to transmit subranges between range coding operations.
 
 b~i~
-: the i-th Range coded binary value.
+: the i-th range-coded binary value.
 
 S~0,\ i~
 : the i-th initial state.
 
 j~n~
 : the length of the bytestream encoding n binary symbols.
 
-The following Range coder state variables are initialized to the following values. The Range is initialized to a value of 65,280 (expressed in base 16 as 0xFF00) as depicted in [@figureInitializeRange]. The Low is initialized according to the value of the first two bytes as depicted in [@figureInitializeLow]. j~i~ tracks the length of the bytestream encoding while incremening from an initial value of j~0~ to a final value of j~n~. j~0~ is initialized to 2 as depicted in [@figureInitializeLength].
+The following range coder state variables are initialized to the following values. The Range is initialized to a value of 65,280 (expressed in base 16 as 0xFF00) as depicted in [@figureInitializeRange]. The Low is initialized according to the value of the first two bytes as depicted in [@figureInitializeLow]. j~i~ tracks the length of the bytestream encoding while incrementing from an initial value of j~0~ to a final value of j~n~. j~0~ is initialized to 2 as depicted in [@figureInitializeLength].
 
 SVGI:!---
 SVGI:![svg](rangebinaryvalues5.svg "range binary values 5")
 SVGI:!---
 SVGC:rangebinaryvalues5.svg=$$R_{0}=65280$$
 AART:R_(0) = 65280
-Figure: The initial value for "Range". {#figureInitializeRange}
+Figure: The initial value for the Range. {#figureInitializeRange}
 
 SVGI:!---
 SVGI:![svg](rangebinaryvalues6.svg "range binary values 6")
 SVGI:!---
 SVGC:rangebinaryvalues6.svg=$$L_{0}=2^{8}B_{0}+B_{1}$$
 AART:L_(0) = 2 ^ 8 * B_(0) + B_(1)
-Figure: The initial value for "Low" is set according to the first two bytes of the bytestream. {#figureInitializeLow}
+Figure: The initial value for Low is set according to the first two bytes of the bytestream. {#figureInitializeLow}
 
 SVGI:!---
 SVGI:![svg](rangebinaryvalues7.svg "range binary values 7")
 SVGI:!---
 SVGC:rangebinaryvalues7.svg=$$j_{0}=2$$
 AART:j_(0) = 2
-Figure: The initial value for "j", the length of the bytestream encoding. {#figureInitializeLength}
+Figure: The initial value for j, the length of the bytestream encoding. {#figureInitializeLength}
```

Comment:
Should it be
```
value for `j`, the length
```
?

Reply:
I don't know why this changed within the AUTH48 process. For this PR, my goal was to sync with AUTH48 so that the changes there are available for ongoing work on version 4.

Reply:
To me it’s the same change that you made a few lines later with `k`, therefore I mentioned it for consistency.